modules/ossm-integrating-with-user-workload-monitoring.adoc

```diff
@@ -0,0 +1,174 @@
+<2> (Optional) Setting `spec.selector.matchLabels` applies the telemetry resource to specific workloads in the target namespace.
++
+
+. Apply a `ServiceMonitor` to monitor the Istio control plane:
++
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: istiod-monitor
+  namespace: istio-system # <1>
+spec:
+  targetLabels:
+  - app
+  selector:
+    matchLabels:
+      istio: pilot
+  endpoints:
+  - port: http-monitoring
+    interval: 30s
+    relabelings:
+    - action: replace
+      replacement: "<SMCP_NAME>-<SMCP_NAMESPACE>" # <2>
+      targetLabel: mesh_id
+----
+<1> Since OpenShift Monitoring ignores the field `spec.namespaceSelector` in `ServiceMonitor` and `PodMonitor` objects, you need to apply the `PodMonitor` in all mesh namespaces, including the control plane namespace.
+<2> A shared Prometheus instance can mean there are metrics from multiple meshes. However, if you are deploying Kiali, the metrics must be associated with a single mesh. To associate metrics with a single mesh, add a unique `mesh_id` to each one. Doing so narrows the `query_scope` in Kiali to only the relevant mesh metrics.
++
+
+. Apply a `PodMonitor` to collect metrics from Istio proxies:
++
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: istio-proxies-monitor
+  namespace: istio-system # <1>
+spec:
+  selector:
+    matchExpressions:
+    - key: istio-prometheus-ignore
+      operator: DoesNotExist
+  podMetricsEndpoints:
+  - path: /stats/prometheus
+    interval: 30s
+    relabelings:
+    - action: keep
+      sourceLabels: [__meta_kubernetes_pod_container_name]
+      regex: "istio-proxy"
+    - action: keep
+      sourceLabels: [__meta_kubernetes_pod_annotationpresent_prometheus_io_scrape]
+    - action: replace
+      regex: (\d+);(([A-Fa-f0-9]{1,4}::?){1,7}[A-Fa-f0-9]{1,4})
+      replacement: '[$2]:$1'
+      sourceLabels: [__meta_kubernetes_pod_annotation_prometheus_io_port, 
+      __meta_kubernetes_pod_ip]
+      targetLabel: __address__
+    - action: replace
+      regex: (\d+);((([0-9]+?)(\.|$)){4})
+      replacement: $2:$1
+      sourceLabels: [__meta_kubernetes_pod_annotation_prometheus_io_port, 
+      __meta_kubernetes_pod_ip]
+      targetLabel: __address__
+    - action: labeldrop
+      regex: "__meta_kubernetes_pod_label_(.+)"
+    - sourceLabels: [__meta_kubernetes_namespace]
+      action: replace
+      targetLabel: namespace
+    - sourceLabels: [__meta_kubernetes_pod_name]
+      action: replace
+      targetLabel: pod_name
+    - action: replace
+      replacement: "<SMCP_NAME>-<SMCP_NAMESPACE>" # <2>
+      targetLabel: mesh_id
+----
+<1> Since OpenShift Monitoring ignores the field `spec.namespaceSelector` in `ServiceMonitor` and `PodMonitor` objects, you need to apply the `PodMonitor` in all mesh namespaces, including the control plane namespace.
```

Comment:
```suggestion
<1> Since OpenShift Monitoring ignores the field `spec.namespaceSelector` in `ServiceMonitor` and `PodMonitor` objects, you need to apply the `PodMonitor` object in all mesh namespaces, including the control plane namespace.
```